src/ui/src/common/extension_command.ts

```diff
@@ -17,19 +17,39 @@
  */
 
 import {Graph, GraphCollection,} from '../components/visualizer/common/input_graph';
-import type { ChangesPerNode, ExecutionCommand } from './model_loader_service_interface';
+import type { NodeDataProviderData } from '../components/visualizer/common/types';
+import type { ChangesPerNode } from './model_loader_service_interface';
 
 /** A command sent to extension. */
 export declare interface ExtensionCommand {
   cmdId: string;
   extensionId: string;
+  modelPath: string;
```

Comment:
So is this now part of every command? Do we need it to be?

Reply:
We are sending it with every command, yes.

I think it is interesting as the model path can be used as a unique identifier if you want to keep any metadata on the server.

Reply:
Every command as part of upstream model-explorer also requires this trait, I think it's useful to have it exposed for our use as well.